### ISSUE TITLE
Missing license information

### ISSUE THREAD

**Issue by nj314** [teambit/bvm#18]
All published npm packages from this repo are currently missing LICENSE information. This means certain organizations cannot use `bvm` because they cannot verify that the license permits their usage.

The following packages are missing license info:

* `@teambit/bvm.commands.config@0.0.20`
* `@teambit/bvm.commands.install@0.0.23`
* `@teambit/bvm.commands.link@0.0.23`
* `@teambit/bvm.commands.list@0.0.21`
* `@teambit/bvm.commands.remove@0.0.21`
* `@teambit/bvm.commands.upgrade@0.0.23`
* `@teambit/bvm.commands.version@0.0.23`
* `@teambit/bvm.config@0.0.20`
* `@teambit/bvm.error@0.0.1`
* `@teambit/bvm.fetch@0.0.22`
* `@teambit/bvm.install@0.0.23`
* `@teambit/bvm.link@0.0.23`
* `@teambit/bvm.list@0.0.21`
* `@teambit/bvm.remove@0.0.21`
* `@teambit/gcp.storage@0.0.17`
* `@teambit/network.agent@0.0.90`
* `@teambit/network.proxy-agent@0.0.90`
* `@teambit/time.time-format@0.0.298`
* `@teambit/toolbox.fs.progress-bar-file-extractor@0.0.1`
* `date-fns@2.19.0` (perhaps out-of-scope for this issue?)
* `@teambit/toolbox.network.progress-bar-file-downloader@0.0.18`

**Comment by GiladShoham** [comment]
That's right, I'll make sure they have a license.
Thanks.

**Comment by ChazUK** [comment]
Hi @GiladShoham, is there any update on this. My organisation has asked me to look into integrating Bit into our workflow, but unfortunately I'm stumped on the first hurdle.

**Comment by ChazUK** [comment]
Thanks @GiladShoham can you confirm that these changes have been published to npm?

**Comment by GiladShoham** [comment]
about these 
```
@teambit/time.time-format@0.0.298:SEE,LICENSE,IN,LICENSE
@teambit/network.proxy-agent@0.0.90:SEE,LICENSE,IN,LICENSE
@teambit/network.agent@0.0.90:SEE,LICENSE,IN,LICENSE
```
It's just about upgrading the versions used by bvm. 
I'll take care of this soon.

About the longer list, those are not used by bvm, they are used by bit itself.
Did you check bvm only or bit as well?

**Comment by ChazUK** [comment]
@GiladShoham `npm i -g @teambit/bvm` now works on our internal systems after being installed via Nexus!

But I'm now seeing some another error with `bvm install`. I'm currently in the process of talking to our it/network guy as we're getting a timeout error

```
request to https://storage.googleapis.com/storage/v1/b/bvm.bit.dev/o?prefix=versions/dev/Darwin/ failed, reason: connect ETIMEDOUT 172.217.168.208:443
```

**Comment by johnschult** [comment]
@mitchdotdeveloper If you already have `npm` or `yarn` proxies configure then you don't need to do it for `bvm` or `bit`. According the bit docs, they use the configured proxy settings from `npm`. However, even when they are set correctly, the command `bvm install` still errors out with:

```
request to https://storage.googleapis.com/storage/v1/b/bvm.bit.dev/o?prefix=versions/dev/Darwin/ failed, reason: unable to get local issuer certificate
```
This does not indicate that the proxy is not set, it is saying that the certificate chain does not include the one used for the proxy. When you said something about `proxy.ca` and `proxy.cert` you were on to something, but wven with those set it doesn't work. Clearly something is not using those settings.

**Comment by GiladShoham** [comment]
@ChazUK @johnschult @mitchdotdeveloper 
Thanks for your comments.
I really recommend opening a new issue about these issues. as it's definitely not related to the original issue.
Also, this issue is closed, so higher chance people will miss these comments as opposed to new open issues.
Once you will open new one, I'll comment there :)

**Comment by johnschult** [comment]
Yeah I noticed that yesterday. I was checking into how node-fetch was getting used. I'll open an issue today.
